src/js/navButtons.js

```diff
@@ -140,12 +104,12 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
         nextButton.html(nextLabel);
         nextButton.addClass(showSelector);
         if (isFirstPanel) {
-            tooltip.close();  // Close the existing tooltip for the back button otherwise it will linger after the back button becomes hidden
-            tooltip.applier.fireChangeRequest({path: "idToContent." + that.backButtonId, type: "DELETE"});
+            that.close();  // Close the existing tooltip for the back button otherwise it will linger after the back button becomes hidden
```

Comment:
Yes in terms of the nature of the issue that close() tries to resolve.

The issue was when the back button is clicked to the first panel, the tooltip for the back button from the previous page still shows. This close() will close it.